### PR TITLE
Notebooks programming

### PR DESCRIPTION
**Pull request by arokem**
These are some IPython notebooks and some minimal data for the one hour beginners workshop on Monday.

### CODE REVIEW

File: beginner-python/001-arrays.ipynb [hunk 1]
```diff
@@ -0,0 +1,767 @@
...
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[2 3 4]\n",
+      "[1 3]\n",
+      "[1 3 5 7 9]\n"
+     ]
+    }
+   ],
+   "source": [
```
Comment (bcipolli) [inline beginner-python/001-arrays.ipynb:182]: I believe this cell is a duplicate of the previous one.

**Comment by bcipolli** [comment]
Might be worth mentioning minimum versions in the --I just got a new mac laptop, and it came with matplotlib v1.3.1--the `style` function in 002 doesn't exist there. That could confuse people who aren't using anaconda...

File: beginner-python/001-arrays.ipynb (hunk 1 above)
Comment (arokem) [inline beginner-python/001-arrays.ipynb:182]: Good catch.

**Comment by arokem** [comment]
New macs now come with MPL preinstalled? Halleluya!

If it really is such a new feature, maybe I should just remove it
altogether. It does make things nicer, but getting into software versions
might be beyond the scope of this presentation.

**Comment by bcipolli** [comment]
:) I also don't get `style` when only importing `matplotlib`; I have to import the `style` sub-package explicitly. This is after upgrading.

Could be that the previous error was not a version error, but an import one...

**Comment by bcipolli** [comment]
All works well for me otherwise. One nitpick-ish comment, not sure if it's useful:
Some of the plots in 002-004 have axis labels, titles, and legends--others do not. My intuition is to either encourage people new to hacking to label everything, always (to communicate with others better and to have a better idea of what you were thinking when you come back to it), or to label nothing (just do fast data exploration, without worrying about any of that stuff).

**Comment by arokem** [comment]
Thanks for the comments @bcipolli - I added some labels here and there. May as well show these lines as often as possible. People will figure out for themselves that they don't need to write these lines :-)

**Pull request merged**